ui/views.py

```diff
@@ -52,10 +54,39 @@ def dashboard(request, *args):  # pylint: disable=unused-argument
     }
 
     return render(
-        request, "dashboard.html",
+        request,
+        "dashboard.html",
         context={
             "style_src": get_bundle_url(request, "style.js"),
             "dashboard_src": get_bundle_url(request, "dashboard.js"),
             "js_settings_json": json.dumps(js_settings),
         }
     )
+
+
+def handler404(request):
```

Comment:
I cannot change the name of `handler404`: this is the standard way to override the django default one.